packages/react-components/src/components/VideoGallery/Layout.ts

```diff
@@ -30,7 +30,7 @@ export interface LayoutProps {
    * Maximum number of participant remote video streams that is rendered.
    * @defaultValue 4
    */
-  maxRemoteVideoStreams?: number;
+  maxRemoteVideoStreams: number;
```

Comment:
Why do we make this required? looks like we have default value to 4 when it is undefined?

Reply:
since it will be always defined in our internal implementation of the video gallery this will always be true. this is me thinking more about what happens should we export these components externally so contoso can build their own layouts using the galleries. this should be required so that we don't hit errors off the calling SDK.

Reply:
Yep, I got the point that maxRemoteVideoStreams must have a value - but we could always do a if else like

const maxStreams = maxRemoteVideoStreams ?? 4;

Then use value instead

The idea is we should minimize props needed for a component to run, especially for some tech details most people don't really care a lot. Either make it default to 4(more lean to ACS) or infinite (more lean to generic component) should work well